### DISCUSSION TITLE
Skip only PV and PVC on restore

### DISCUSSION THREAD

**Discussion by lkalaivanan** [vmware-tanzu/velero#5841] [Community support Q&A]
I took my cluster backup with following command,

velero backup create dent-crd  --include-cluster-resources=true

on restore i need to do one NS per time and also exclude pv and pvc if any, so i am doing below,

velero restore create --from-backup dent  --include-namespaces <NS1> --include-cluster-resources=true --exclude-resources pv,pvc
velero restore create --from-backup dent  --include-namespaces <NS2> --include-cluster-resources=true --exclude-resources pv,pvc

Questions:
- Is there any optimal way to do it.
- Also after restoring due to some CRD / webhook config issue the envoy proxies are not injected properly.

**Comment by draghuram** [comment]
Is there any particular reason you are including cluster resources in restore though you want to restore only one namespace at a time?
